cli/scale.go

```diff
@@ -91,7 +91,12 @@ func runScale(args *docopt.Args, client *controller.Client) error {
 		if err != nil {
 			fmt.Println(commands["scale"].usage)
 		}
-		processes[arg[:i]] = val
+		processType := arg[:i]
+		if _, ok := release.Processes[processType]; ok {
+			processes[processType] = val
+		} else {
+			fmt.Printf("No process type exists with name \"%s\".\n", processType)
```

Comment:
OK so do what I did in the controller? Also, I don't see a reason not to succeed with the valid process types. So in your above example, it could be:
```
$ flynn scale web=1 workr=2 foo=3
WARNING: unknown process types: "workr", "foo"
scaling web 0=>1
```